app/src/main/res/values/strings.xml

```diff
@@ -59,6 +59,9 @@
     <string name="select_action">Always Show Game Information</string>
     <string name="select_action_desc_on">Game information will be shown on clicking a game</string>
     <string name="select_action_desc_off">Game information will only be shown on long-clicking a game</string>
+    <string name="filter_invalid_files">Hide Invalid Files</string>
+    <string name="filter_invalid_files_desc_off">Show files that have been parsed as invalid</string>
```

Comment:
Would probably drop this string and only keep one for the description.

Reply:
I didn't like the title, so that works.